actions/webhook.go

```diff
@@ -103,6 +104,36 @@ func (w *Webhook) Run(ctx context.Context, record graveler.HookRecord, writer *H
 	return nil
 }
 
+func writeOutput(ctx context.Context, writer *HookOutputWriter, req *http.Request, resp *http.Response, reqBody []byte) error {
```

Comment:
https://golang.org/pkg/net/http/httputil
I was thinking we should use dump request and response